projects/Mallard/src/components/layout/animators/clipFromTop.tsx

```diff
@@ -87,13 +87,15 @@ const MaskClipFromTop = ({ children, from, easing }: PropTypes) => {
 
 const ClipFromTop = ({ children, from, easing }: MaybePropTypes) => {
     /* android struggles animating masks sadface */
-    if (!supportsAnimatedClipView() || from == undefined || easing == undefined)
+    if (supportsAnimatedClipView() && from && easing) {
+        return (
+            <MaskClipFromTop from={from} easing={easing}>
+                {children}
+            </MaskClipFromTop>
+        )
+    } else {
         return <>{children}</>
-    return (
-        <MaskClipFromTop from={from} easing={easing}>
-            {children}
-        </MaskClipFromTop>
-    )
+    }
 }
```

Comment:
I've changed this conditioal to cover the case where `from` is equal to `NaN`, this was happening after multiple rotations and therefore led to a masked view covering the whole screen after dismissing lightbox.